tools/topology/topology2/development/tplg-targets.cmake

```diff
@@ -81,6 +81,68 @@ PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-sof-hda-efx-generic-4ch.bin,\
 EFX_FIR_PARAMS=passthrough,EFX_IIR_PARAMS=passthrough,\
 EFX_DRC_COMPONENT=multiband,EFX_MBDRC_PARAMS=passthrough"
 
+# With 16 kHz DMIC1
+"sof-hda-generic\;sof-hda-generic-4ch-48k-16k\;\
```

Comment:
I think it might consider namespacing the development topologies for different DSP architectures.
If I'm not mistaken, these are for CAVS2.5 and will not work on ACE1/2?

I have a PR (https://github.com/thesofproject/sof/pull/8712#) to merge the production topologies and it introduces namespacing for hda-generic topologies:
topologies w/o NHLT (generic):
```
sof-hda-generic.tplg
sof-hda-generic-2ch.tplg
sof-hda-generic-4ch.tplg
```
With CAVS2.5 NHLT:
```
sof-hda-generic-cavs25-2ch.tplg
sof-hda-generic-cavs25-4ch.tplg
```
With ACE1/2 NHLT:
```
sof-hda-generic-ace1-2ch.tplg
sof-hda-generic-ace1-4ch.tplg
```
Probably introduce similar naming right now or later when #8712 is merged?

Reply:
Yep, your naming change looks very good to me. I've been wondering how I could differentiate TGL and MTL tplg builds. I will do this update.